samza-sql/src/main/java/org/apache/samza/sql/data/SamzaSqlRelMessage.java

```diff
@@ -32,14 +34,17 @@
  * SamzaSqlRelationalMessage, In future if we find a need, we could add additional column ddl metadata around
  * primary Key, nullability, etc.
  */
-public class SamzaSqlRelMessage {
+public class SamzaSqlRelMessage implements Serializable {
 
   public static final String KEY_NAME = "__key__";
 
-  private final List<Object> fieldValues = new ArrayList<>();
-  private final List<String> fieldNames = new ArrayList<>();
   private final Object key;
 
+  @JsonProperty("fieldNames")
+  private final List<String> fieldNames;
+  @JsonProperty("fieldValues")
+  private final List<Object> fieldValues;
```

Comment:
Can you add a TODO for the workitem to support serialization for the nested SamzaSQLRelMessages?